charts/posthog/templates/decide-deployment.yaml

```diff
@@ -170,6 +170,5 @@ spec:
         securityContext: {{- omit .Values.decide.securityContext "enabled" | toYaml | nindent 12 }}
         {{- end }}
       initContainers:
-      {{- include "_snippet-initContainers-wait-for-service-dependencies" . | indent 8 }}
-      {{- include "_snippet-initContainers-wait-for-migrations" . | indent 8 }}
```

Comment:
Decide still makes queries to PostgreSQL right? So we still need to wait for them to have been applied?

Reply:
I think I might have misunderstood earlier & mislead the ff team ... this would only apply on new container setup, not on restart? If that's true and considering we do rolling deploys this is fine to keep as is.

Reply:
Yeah if you don't include the migration check, you'll end up with a `SELECT all, the, fields, including, ones, that, don't, exist, yet FROM posthog_persons` which will error and if we're lucky fall back to redis, or just straight error out. Either way I don't think that's desired.

Reply:
just confirmed that init containers only run when the pod is created

pods never restart, only the containers within them